architecture/modules/service/main.tf

```diff
@@ -0,0 +1,52 @@
+resource "google_cloud_run_service" "default" {
```

Comment:
We absolutely should have all the cloud architecture documented. Kind of slack here not doing things incrementally, but my plan is (I can already see you shaking your head btw) push an MVP out for this and the transcriber, and then focus intensely on documentation and  increasing coverage. 

Here I was thinking that this module would be used for both the trainer and transcriber, since they'll both take job requests from a pubsub queue. This + the reason that the cloud run service is namespaced within the module made me rename this from 'trainer' back to 'default' since I thought there will only be one cloud run service inside here.

Reply:
> Here I was thinking that this module would be used for both the trainer and transcriber, since they'll both take job requests from a pubsub queue. This + the reason that the cloud run service is namespaced within the module made me rename this from 'trainer' back to 'default' since I thought there will only be one cloud run service inside here.

Sounds fair enough to me.

> We absolutely should have all the cloud architecture documented. Kind of slack here not doing things incrementally, but my plan is (I can already see you shaking your head btw) push an MVP out for this and the transcriber, and then focus intensely on documentation and increasing coverage.

MVPs are fine, but you do have to make sure that you actually come back, sooner rather than later. The bus factor on this project is small and I'm sure neither Ben nor I want to have to understand this from scratch if that bus (or job offer, or whatever) ever did come.